lib/db.js

```diff
@@ -302,6 +305,31 @@ module.exports = function (
       )
   }
 
+  DB.prototype.sessionTokenVerified = function (id) {
```

Comment:
ref naming discussion in https://github.com/mozilla/fxa-auth-db-mysql/pull/132/files#r60721355